src/shader/layerShader.js

```diff
@@ -0,0 +1,52 @@
+export class LayerShader {
+    static get LineVertexSource() {
```

Comment:
According to our programming guide, we should use `lineVertexSource`  instead of `LineVertexSource`.